src/sentry/static/sentry/app/routes.jsx

```diff
@@ -879,7 +878,12 @@ function routes() {
           <Route path="issues/" component={errorHandler(Stream)} />
 
           <Route path="searches/:searchId/" component={errorHandler(Stream)} />
-          <Route path="dashboard/" component={errorHandler(ProjectDashboard)} />
+          <Route
+            path="dashboard/"
+            componentPromise={() =>
+              import(/*webpackChunkName:"ProjectDashboard"*/ './views/projectDashboard')}
```

Comment:
Please add spaces like #11348

Reply:
Is that interpreted differently?

Reply:
No, but I'm just trying to make them consistent.